gatsbyjs/package.json

```diff
@@ -8,25 +8,28 @@
     "gatsby"
   ],
   "scripts": {
-    "develop": "gatsby develop",
+    "develop": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop",
     "start": "gatsby develop",
     "build": "gatsby build",
     "clean": "gatsby clean"
   },
   "license": "0BSD",
   "dependencies": {
+    "@storyblok/js": "^1.0.4",
```

Comment:
Do we need to install the `@storyblok/js` package? `@storyblok/react` contains most of its functionality

Reply:
@fgiuliani Yes, for Gatsby, we need to. The reason is that Gatsby returns `content` with string and we need to include a workaround in a custom hook.

That's what we need to include in gatsby-source-storyblok soon.

[https://github.com/schabibi1/getting-started/blob/update-gatsby-repo/gatsbyjs/src/utils/storyblok-service.js#L6](https://github.com/schabibi1/getting-started/blob/update-gatsby-repo/gatsbyjs/src/utils/storyblok-service.js#L6)

Reply:
probably I'm missing something, but what's the difference between the `useStoryblokBridge` function from `@storyblok/js` and the one exposed by `@storyblok/react`? 🤔 

https://github.com/storyblok/storyblok-react/blob/feature/v1/lib/index.js#L11

Reply:
@fgiuliani I tested and you're right, I missed that we expose the same `useStoryblokBridge` 

I think I was using `sbBridge` and it threw me an error and it showed me the error of the place to define custom hooks.
Fixed it and will push it after checking all the following comments.